packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -238,7 +236,11 @@ const browsers = computed(() => {
 const setBrowser = useMutation(OpenBrowserList_SetBrowserDocument)
 
 const selectedBrowserId = computed({
-  get: () => props.gql.currentBrowser?.id || props.gql.browsers?.find((browser) => browser.displayName === 'Electron')?.id,
+  get: () => {
+    // NOTE: The currentBrowser is set to the first detected browser
+    // found during project initialization. It should always be defined.
+    return props.gql.currentBrowser?.id
+  },
```

Comment:
I don't think the defaulting value logic was ever executed outside of component tests, as we always default the currentBrowser to the first browser detected on the user's system, which would always at least include Electron. So I removed that bit.

I also updated the markup so that if we end up in a scenario where we don't have a currentBrowser, we do not show the action buttons and allow the user to open a browser when nothing is actually selected.